src/Sylius/Behat/Page/Shop/Checkout/CompletePage.php

```diff
@@ -291,6 +297,25 @@ public function getShippingPromotionDiscount($promotionName)
     /**
      * {@inheritdoc}
      */
+    public function tryToOpen(array $urlParameters = [])
+    {
+        if ($this->getDriver() instanceof Selenium2Driver) {
+            $start = microtime(true);
+            $end = $start + 15;
+            do {
+                parent::tryToOpen($urlParameters);
+                sleep(1);
+            } while(!$this->isOpen() && microtime(true) < $end);
```

Comment:
It will try to open page then it will wait 1s. If is not opened and actual time is less than timeout. It will repeat.